### PR TITLE
Add margin and color changes

### LINKED ISSUE #153: Create requisition modal
The CreateRequisitionModal has a few errors:

- active-background of the drop down cuts through a part of the top border.
- The order type label of the drop down should be orange to be consistent (Or select program label grey)
- Both labels, Select Program and Order Type should be aligned correctly.

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #153

Minor changes to a couple of margins/padding's and colors

### CODE REVIEW

File: src/ui/components/Modals/CreateRequisitionModal.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ class CreateRequisitionModal extends React.PureComponent {
           options={orderTypes}
           selected={selectedOrderTypeId}
           changeHandler={this.selectionHandler('selectedOrderTypeId')}
-          style={{ paddingLeft: 30, paddingRight: 30 }}
+          style={{ paddingLeft: 30, paddingRight: 30, marginBottom: 20, marginTop: 20 }}
```
Comment (wlthomson) [inline src/ui/components/Modals/CreateRequisitionModal.js]: Could we make a `localStyles` for this?

File: src/ui/components/Modals/CreateRequisitionModal.js [hunk 2]
```diff
@@ -113,9 +113,10 @@ class CreateRequisitionModal extends React.PureComponent {
           dataLabel="name"
           dataValue="id"
           height={150}
+          style={{ paddingLeft: 30, paddingRight: 30 }}
         />
 
-        <div style={{ display: 'flex', justifyContent: 'center' }}>
+        <div style={{ display: 'flex', justifyContent: 'center', margin: 'auto' }}>
```
Comment (wlthomson) [inline src/ui/components/Modals/CreateRequisitionModal.js]: As above, `localStyles` would be good (perhaps a pure component, e.g. `ButtonGroup`).

**Comment by wlthomson** [comment]
Looks good. A few style comments above.

File: src/ui/components/Modals/CreateRequisitionModal.js (hunk 1 above)
Comment (joshxg) [reply src/ui/components/Modals/CreateRequisitionModal.js]: cool, done

**Comment by wlthomson** [comment]
Looks good. Merging and closing.

**Pull request merged**